java-shared-dependencies/dependency-analyzer/pom.xml

```diff
@@ -38,6 +38,16 @@
           </arguments>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.2.5</version>
+        <configuration>
+          <environmentVariables>
+            <GITHUB_TOKEN>fake_value</GITHUB_TOKEN>
```

Comment:
Add source code comment how it helps tests.

Reply:
Done.